backend/app.py

```
@@ -0,0 +1,375 @@
+    if row is None:
+        return jsonify({"error": f"Event {event_id} not found"}), 404
+
+    db.execute("DELETE FROM events WHERE id = ?", (event_id,))
+    db.commit()
+    return jsonify({"message": f"Event {event_id} deleted."}), 200
+
+
+# ── Entry point ───────────────────────────────────────────────────────────────
+if __name__ == "__main__":
+    init_db()   # creates events.db and seeds sample data on first run
+    app.run(host="0.0.0.0", port=3000, debug=True)
```

Ruff (0.15.1)

[error] Possible binding to all interfaces

(S104)